src/store/conversationsStore.js

```diff
@@ -102,6 +102,27 @@ const actions = {
 		if (!currentUser.uid) {
 			currentUser = getCurrentUser()
 		}
+
+		if (!currentUser || !currentUser.uid) {
```

Comment:
add comment to clarify that for guest users we need to explicitly fetch the list (the if statement isn't very clear that we're targetting guests)